admin/config.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
   CHECK_FORM_NAME_MAX_CHARACTERS: 128,
   CHECK_FORM_MIN_INTEGER: 1,
   CHECK_FORM_MAX_INTEGER: 12,
-  LINES_PER_CHECK_FORM: process.env.LINES_PER_CHECK_FORM || '25',
+  LINES_PER_CHECK_FORM: parseInt(process.env.LINES_PER_CHECK_FORM) || 25,
```

Comment:
this will never return the default, because parseInt returns the parsed integer or `NaN`

Reply:
`function getFormLines() {
    const defaultValue = 25
    const val = process.env.LINES_PER_CHECK_FORM
    const parsed = parseInt(val)
    if (parsed === 'NaN') return defaultValue
    if (parsed < 1) return defaultValue
    return parsed
}`